portsyncd/portsyncd.cpp

```diff
@@ -85,7 +86,11 @@ int main(int argc, char **argv)
         netlink.registerGroup(RTNLGRP_LINK);
         cout << "Listen to link messages..." << endl;
 
-        handlePortConfigFile(p, port_config_file);
+        if (port_config_file.empty()) {
+            handlePortConfigFile(p, port_config_file);
```

Comment:
use the port_config_file when it is empty?

Reply:
`string::empty()` returns 0 if the string is empty

http://www.cplusplus.com/reference/string/string/empty/